frame/support/src/lib.rs

```diff
@@ -1234,7 +1234,10 @@ pub mod pallet_prelude {
 		EqNoBound, PartialEqNoBound, RuntimeDebugNoBound, DebugNoBound, CloneNoBound, Twox256,
 		Twox128, Blake2_256, Blake2_128, Identity, Twox64Concat, Blake2_128Concat, ensure,
 		RuntimeDebug, storage,
-		traits::{Get, Hooks, IsType, GetPalletVersion, EnsureOrigin, PalletInfoAccess},
+		traits::{
+			Get, Hooks, IsType, GetPalletVersion, EnsureOrigin, PalletInfoAccess, StoragesInfo,
+			ConstU32, GetDefault,
```

Comment:
Looks like `ConstU32` isn't actually used in this file?

Reply:
this is the prelude that people ususally import when declaring their pallet.

ConstU32 can be used to declare MaxValues easily:
```rust
type Foo<T> = StorageMap<_, Twox64Concat, u32, u32, OptionQuery, GetDefault, ConstU32<100_000>>
```